src/text-area/index.tsx

```diff
@@ -196,6 +201,60 @@ export const TextArea = factory(function TextArea({
 	const [labelChild] = children();
 	const label = isRenderResult(labelChild) ? labelChild : labelChild.label;
 
+	let {
+		size: { width: labelWidth }
+	} = dimensions.get('label');
+
+	// When notched the label is transformed in the CSS to 75% size, then we need to add 10px of padding to get the desired look
+	labelWidth = labelWidth * 0.75 + 10;
```

Comment:
is this the same with dojo theme too?

Reply:
Sadly no. Is there a way to detect the theme name? I did not see a way. I got around this by using CSS for override the width of the label one the dojo theme with an `important!` tag.

Reply:
Can't you just override the transform in the material theme CSS?

Reply:
Are you suggesting removing the 75% text size transform from the material theme?

Reply:
not necessarily, I'm just saying that you should be doing this adjustment, whatever it needs to be, in the material theme css and not in this widget js.